samples/show-scale-bar/src/main/java/com/esri/arcgismaps/sample/showscalebar/screens/ShowScaleBarScreen.kt

```diff
@@ -0,0 +1,245 @@
+                )
+                Scalebar(
+                    modifier = Modifier
+                        .padding(horizontal = 25.dp, vertical = 50.dp)
+                        .align(Alignment.BottomStart),
+                    maxWidth = 300.dp,
+                    unitsPerDip = mapViewModel.unitsPerDip,
+                    viewpoint = mapViewModel.viewpoint,
+                    spatialReference = mapViewModel.spatialReference,
+                    style = currentScalebarStyle,
+                    units = currentScalebarUnitSystem,
+                    autoHideDelay = currentAutoHideDelay,
+                    useGeodeticCalculations = isGeodeticCalculationsEnabled
+                )
+
+                if (isScalebarDialogOptionsVisible) {
+                    ScalebarDialogOptions(
+                        currentScalebarStyle = currentScalebarStyle,
+                        currentScalebarUnitSystem = currentScalebarUnitSystem,
+                        currentAutoHideDelay = currentAutoHideDelay,
+                        isGeodeticCalculationsEnabled = isGeodeticCalculationsEnabled,
+                        onScalebarUnitSystemSelected = { currentScalebarUnitSystem = it },
+                        onScalebarStyleSelected = { currentScalebarStyle = it },
+                        onDismissRequest = { isScalebarDialogOptionsVisible = false },
+                        onAutoHideDelaySelected = { currentAutoHideDelay = it },
+                        onGeodeticCalculationsToggled = {
+                            isGeodeticCalculationsEnabled = !isGeodeticCalculationsEnabled
+                        }
+                    )
+                }
+            }
+            mapViewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    )
+}
+
+@Composable
+fun ScalebarDialogOptions(
+    currentScalebarStyle: ScalebarStyle,
+    currentScalebarUnitSystem: UnitSystem,
+    currentAutoHideDelay: Duration,
+    onDismissRequest: () -> Unit,
+    isGeodeticCalculationsEnabled: Boolean,
+    onScalebarStyleSelected: (ScalebarStyle) -> Unit,
+    onScalebarUnitSystemSelected: (UnitSystem) -> Unit,
+    onAutoHideDelaySelected: (Duration) -> Unit,
+    onGeodeticCalculationsToggled: (Boolean) -> Unit
+) {
+    // List of all the supported scale bar properties
+    val scalebarStyles = ScalebarStyle.entries.toList()
+    val scalebarUnitSystems = listOf(UnitSystem.Metric, UnitSystem.Imperial)
+    val autoHideDelays = listOf(
+        Duration.INFINITE,
+        1.toDuration(DurationUnit.SECONDS),
+        3.toDuration(DurationUnit.SECONDS),
+        5.toDuration(DurationUnit.SECONDS)
```

Comment:
```suggestion
        1.seconds,
        3.seconds,
        5.seconds
```

just a bit of syntactic sugar